systers_portal/users/migrations/0002_usersetting.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.6 on 2020-07-16 08:02
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('users', '0001_squashed_0003_auto_20160207_1550'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='UserSetting',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('weekly_digest', models.BooleanField(default=False)),
```

Comment:
I think Weekly digest should be true by default.

Reply:
Sure, I'll do that

Reply:
Done